src/Compilers/Core/MSBuildTask/Microsoft.Managed.EditorConfig.targets

```diff
@@ -9,8 +9,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PotentialEditorConfigFiles Include="@(Compile->GetPathsOfAllDirectoriesAbove()->Combine('.editorconfig'))" Condition="'$(DiscoverEditorConfigFiles)' != 'false'" />
+    <AllDirectoriesAbove Include="@(Compile->GetPathsOfAllDirectoriesAbove())" Condition="'$(DiscoverEditorConfigFiles)' != 'false'" />
```

Comment:
Usually internal names are prefixed with an _ to avoid collisions with other stuff.